src/fusor/fusion_matching.py

```diff
@@ -118,29 +119,32 @@ def _match_fusion_structure(
         | MultiplePossibleGenesElement
         | GeneElement,
         is_five_prime_partner: bool,
-    ) -> int | str:
+    ) -> int | None:
         """Compare transcript segments for an assayed and categorical fusions
-        :param assayed_element: The assayed fusion transcript or unknown gene element or gene element
-        :param categorical_element: The categorical fusion transcript or mulitple possible genes element
+        :param assayed_element: The assayed fusion transcript or unknown gene element
+            or gene element
+        :param categorical_element: The categorical fusion transcript or mulitple
+            possible genes element
         :param is_five_prime_partner: If the 5' fusion partner is being compared
         :return A score indiciating the degree of match or "NA" if the partner is
             ? or v
         """
         # Set default match score
         match_score = 0
 
-        # If the assayed partner is unknown or the categorical partner is a multiple possible gene element, return match score of 0 as no precise information
+        # If the assayed partner is unknown or the categorical partner is a multiple
+        # possible gene element, return match score of 0 as no precise information
         # regarding the compared elements is known
         if isinstance(assayed_element, UnknownGeneElement) or isinstance(
             categorical_element, MultiplePossibleGenesElement
         ):
-            return "NA"
+            return None
 
         # Compare gene partners first
         if assayed_element.gene == categorical_element.gene:
-            match_score += 1
+            match_score += 1  # GeneElement objects match
```

Comment:
I'm not sure these are super helpful since you have a top-level comment for each section. What my suggestion was getting at was to update the docstring `Compare transcript segments for an assayed and categorical fusions` to include a little more information